### DISCUSSION TITLE
Dreambooth local vs Dreambooth Colab

### DISCUSSION THREAD

**Discussion by DrmeddieWiese** [TheLastBen/fast-stable-diffusion#1083] [Q&A] [answered]
Hi  together,

I have always trained my models in the Colab version of Dreambooth. Now I am locally in Automatic1111 on the road and there it unfortunately does not work so well with the models.

Can someone tell me which parameters are set in the Colab version? I had already tried to read the code. But in vain.

For example, is EMA used in the Colab version, which scheduler is selected, are sample images created, etc.?

My locally trained model is unfortunately never as usable as in Colab?

Thank you very much.

**Comment by TheLastBen** [comment, marked as answer]
the shceduler is "polynomial" and the learning rate is set to 2e-6, those are the main differences, add to that the ability to train the unet and the text_enc separately

**Reply by DrmeddieWiese** [reply]
Thanks. But what do you mean with ability to train unet and Text encoder separately? I have trained the Text encoder separately. But I don't know where to train unet

**Reply by TheLastBen** [reply]
simply by removing the flag train_text_encoder, you'd be training the unet only
